modules/objdetect/include/opencv2/objdetect.hpp

```diff
@@ -693,23 +693,43 @@ class CV_EXPORTS_W QRCodeDetector
      */
     CV_WRAP bool detect(InputArray img, OutputArray points) const;
 
+    /** @brief Detects QR codes in image and returns the vector of the quadrangle containing the codes.
+     @param img grayscale or color (BGR) image containing (or not) QR codes.
+     @param points Output vector of vector of vertices of the minimum-area quadrangle containing the codes.
+     */
+    CV_WRAP bool multipleDetect(InputArray img, OutputArrayOfArrays points) const;
```

Comment:
detectMulti (like `detectMultiScale()` in objdetect)

From API point of view - it make sense to add `maxCount` or `minSize` parameters. Or add them as a class object properties.